geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/ManagementControllerAdvice.java

```diff
@@ -45,8 +45,8 @@
   @ExceptionHandler(EntityExistsException.class)
   public ResponseEntity<ClusterManagementResult> entityExists(final Exception e) {
     // no need to log the error stack. User only needs to know the message.
-    return new ResponseEntity<>(new ClusterManagementResult(false, e.getMessage()),
-        HttpStatus.CONFLICT);
+    return new ResponseEntity<>(new ClusterManagementResult(true, e.getMessage()),
+        HttpStatus.OK);
```

Comment:
This change should be reverted.

Reply:
As the result of the internal API, we only need a way to indicate that this entity already exists when creating it the 2nd time, throwing an exception is a very clear way to indicate that. It's up to the caller of this internal API to interpret this exception depending on different "style" of REST.

Reply:
Do we really need a 3rd status "NO_OP" in the ClusterManagementResult?